Apps/JobScheduler/src/Tasks/CopyEmailToMessagingVerification.cs

```diff
@@ -81,13 +81,15 @@ public async Task RunAsync(CancellationToken ct = default)
                 await Task.WhenAll(tasks);
 
                 await this.dbContext.SaveChangesAsync(ct);
-                this.logger.LogInformation("Saved message verification changes after {Processed} email(s) processed", processed);
 
                 emails = await query.Take(this.batchSize).ToListAsync(ct);
-                this.logger.LogInformation("The number of emails to copy from Email.To to MessagingVerification.EmailAddress: {Emails}", emails.Count);
+                this.logger.LogInformation(
+                    "Saved message verification changes after {Processed} email(s) processed. \n The number of emails copied from Email.To to MessagingVerification.EmailAddress: {Emails}",
+                    processed,
+                    emails.Count);
```

Comment:
Suggest removing whitespace around linebreak:

```suggestion
                this.logger.LogInformation(
                    "Saved message verification changes after {Processed} email(s) processed.\nThe number of emails copied from Email.To to MessagingVerification.EmailAddress: {Emails}",
                    processed,
                    emails.Count);
```